lib/crypto.js

```diff
@@ -633,6 +633,37 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
+function randomFillSync(buf, offset = 0, size) {
+  if (!Buffer.isBuffer(buf)) {
+    throw new TypeError('"buf" argument must be a Buffer');
```

Comment:
Can I suggest that you use `process.binding('util').isUint8Array` instead of `Buffer.isBuffer` and adjust the error message to include `Uint8Array`s?